src/mongo/s/commands/cluster_count_cmd.cpp

```diff
@@ -47,9 +47,9 @@
 namespace mongo {
 namespace {
 
-class ClusterCountCmd : public ErrmsgCommandDeprecated {
+class ClusterCountCmd : public Command {
```

Comment:
I think this should inherit from `BasicCommand` rather than directly from `Command`. This is a `BasicCommand` in the sense that it does not support document sequences. (Batch write commands like insert, update, and delete are examples of commands that accept a sequence of documents as input.) Therefore, this could benefit from the simplified `run()` signature provided by `BasicCommand`